lnwallet/channel.go

```diff
@@ -3145,6 +3172,55 @@ func (lc *LightningChannel) fetchParent(entry *paymentDescriptor,
 	return addEntry, nil
 }
 
+// balanceAboveReserve checks if the balance for the provided party is above the
+// configured reserve. It also uses the balance delta for the party, to account
+// for entry amounts that have been processed already.
+func (lc *LightningChannel) balanceAboveReserve(party lntypes.ChannelParty,
```

Comment:
This doesn't need to be a method. Or we move the `c := lc.channelState` into here, from `evaluateNoOpHtlc`.